ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -204,19 +213,23 @@ def install_internal_cluster(self):
 
         """
         logger.info("Click Internal")
-        self.do_click(locator=self.dep_loc["internal_mode"], enable_screenshot=True)
+        if self.operator == ODF_OPERATOR:
+            self.do_click(
+                locator=self.dep_loc["internal_mode_odf"], enable_screenshot=True
+            )
+        else:
+            self.do_click(locator=self.dep_loc["internal_mode"], enable_screenshot=True)
 
         logger.info("Configure Storage Class (thin on vmware, gp2 on aws)")
         self.do_click(
             locator=self.dep_loc["storage_class_dropdown"], enable_screenshot=True
         )
         self.do_click(locator=self.dep_loc[self.storage_class], enable_screenshot=True)
 
-        device_size = str(config.ENV_DATA.get("device_size"))
-        osd_size = device_size if device_size in osd_sizes else "512"
-        logger.info(f"Configure OSD Capacity {osd_size}")
-        self.choose_expanded_mode(mode=True, locator=self.dep_loc["osd_size_dropdown"])
-        self.do_click(locator=self.dep_loc[osd_size], enable_screenshot=True)
+        if self.operator == ODF_OPERATOR:
+            self.do_click(locator=self.dep_loc["next"], enable_screenshot=True)
```

Comment:
Before clicking on Next, please verify if the Full Deployment is selected as Deployment type from the Advanced option on the Backing storage page. If not, select it.

Reply:
the Full deployment is the default.